contracts/upgradeability/AdminUpgradeabilityProxy.sol

```diff
@@ -4,32 +4,30 @@ import './UpgradeabilityProxy.sol';
 
 /**
  * @title AdminUpgradeabilityProxy
- *
  * @dev This contract combines an upgradeability proxy with an authorization
- * @dev mechanism for administrative tasks.
- *
- * @dev All external functions in this contract must be guarded by the
- * @dev ifAdmin modifier. See ethereum/solidity#3864 for a Solidity
- * @dev feature proposal that would enable this to be done automatically.
+ * mechanism for administrative tasks.
+ * All external functions in this contract must be guarded by the
+ * `ifAdmin` modifier. See ethereum/solidity#3864 for a Solidity
+ * feature proposal that would enable this to be done automatically.
  */
 contract AdminUpgradeabilityProxy is UpgradeabilityProxy {
   /**
-   * @dev Event to show administration has been transferred
-   * @param previousAdmin the address of the previous admin
-   * @param newAdmin the address of the new admin
+   * @dev Emitted when the administration has been transferred.
+   * @param previousAdmin Address of the previous admin.
+   * @param newAdmin Address of the new admin.
    */
   event AdminChanged(address previousAdmin, address newAdmin);
 
   /**
    * @dev Storage slot with the admin of the contract.
-   * @dev This is the hash of "org.zeppelinos.proxy.admin", and is validated in
-   * @dev the constructor.
+   * This is the keccak-256 hash of "org.zeppelinos.proxy.admin", and is
+   * validated in the constructor.
    */
   bytes32 private constant ADMIN_SLOT = 0x10d6a54a4754c8869d6886b5f5d7fbfa5b4522237ea5c60d11bc4e7a1ff9390b;
 
   /**
-   * @dev Will run this function if the sender is the admin.
-   * @dev Otherwise it will fall back to the implementation.
+   * @dev Modifier to ensure that the sender is the admin.
```

Comment:
Doesn't really ensure, more checks.. because something *will* execute if it's not. Suggestion:

"Modifier to check whether the msg.sender is the admin.
 If it is, it will run the function. Otherwise, it will delegate the call to the implementation."

Reply:
perfect.